src/help/guides/getting-started.md

```diff
@@ -63,42 +64,42 @@ This requires iOS 13.3+ or iPadOS 13.4+, the app will CRASH otherwise. Paperback
+1. Restart your PC after you've installed both **iTunes** and **iCloud**.
    <el-tag type="warning">You need to restart your computer!</el-tag>
 1. Extract the downloaded **`AltInstaller.zip`** file. Run **`setup.exe`** and follow the instructions.
-1. Launch **AltServer** from your *Notification Area*/*System Tray* or *Hidden Icons*.
+1. Launch **AltServer** from your *System tray* or *Hidden icons*.
 	> Can't find **AltServer**? [Image](https://imgur.com/a/rSagfh2)
 1. Connect your **iPhone** or **iPad** to your computer and make sure it's unlocked and your device is trusted.
-1. Open **iTunes** and enable "**iTunes Wi-Fi sync**" for your phone.
-1. Click the **AltServer** icon and click "**Install AltStore**, then choose your device.
-1. Enter your Apple ID email and password. (**BOTH** email address and password are case sensitive).
-   > *Your Apple ID and password are sent to Apple, and nowhere else.*
-1. AltStore should then be installed into your phone.
+1. Open **iTunes** and enable "**iTunes Wi-Fi sync**" for your iPhone/iPad.
+1. Click the **AltServer** icon in your *System tray* and click "**Install AltStore**, then select your iPhone/iPad.
+1. Enter your Apple ID email and password. (**BOTH** are case sensitive)
+	> *Your Apple ID and password are sent to Apple, and nowhere else.*
+1. AltStore should be installed to your iPhone/iPad.
 
 ### Paperback
 #### Installation
 1. Open **AltServer** along with **iCloud** and **iTunes**.
 1. Make sure you're on the _same connection_ (same router, in most cases) as the computer running **AltServer**. If you're not connected to the same network, you can connect your device via cable.
-1. Trust AltStore application: on your device go in Settings, General, Profiles & Device Management. Choose your Apple ID and select **Trust**.
-1. Launch **AltStore** on your device.
+1. Trust AltStore application: On your device, go to Settings > General > Device Management. Choose your Apple ID and select **Trust**.
+1. Launch **AltStore** on your iPhone/iPad.
 1. Go to **My Apps** and tap the **+** icon in the top left corner.
-1. Open the **`.IPA`** you downloaded from the [prerequisites](/help/guides/getting-started/#prerequisites) section.
+1. Open the **`.IPA`** you downloaded from the [Prerequisites](/help/guides/getting-started/#prerequisites) section.
 1. **Paperback** should finally be installed!
 :::
 ::::
 
 ## Installation FAQ
 ### I get "Access Denied" when trying to install AltStore
-Run AltServer as an administrator, and ensure that you've restarted after installing iTunes and iCloud.
- 
+Run AltServer as an administrator and ensure that you've restarted your computer after installing iTunes and iCloud.
+
 ### I can't find the .IPA file to download
-Navigate down toward the assets drop down, the .IPA should be listed there. [Image](https://imgur.com/a/onrwNC8)
- 
+Scroll down to the **Assets** list, the .IPA should be listed there. [GIF](https://imgur.com/a/onrwNC8)
```

Comment:
Does this step still makes sense? I think we removed all instructions asking to download the IPA through GitHub.

Reply:
Yeah we got the fancy button now that goes to the latest .ipa right away. I'll link this in this instruction.

Reply:
Fixed in d2a4a6fc39706f5154330cb130e3d8ad2f099314